modules/transactions/src/test/java/org/apache/ignite/internal/tx/HeapLockManagerTest.java

```diff
@@ -133,4 +135,13 @@ public void testNonDefaultConfiguration(
         assertThat(lockManager.available(), is(42));
         assertThat(lockManager.getSlots(), is(arrayWithSize(0)));
     }
+
+    @Test
+    public void test() {
```

Comment:
please rename the test method

Reply:
I didn't expect such output..

<img width="1162" alt="Screenshot 2025-03-11 at 10 57 07" src="https://github.com/user-attachments/assets/5b7d1146-051a-49d4-8e40-d54587271ac0" />